src/cli_plugin/list/list.js

```diff
@@ -1,13 +1,28 @@
-import { statSync, readdirSync } from 'fs';
+import { statSync, readdirSync, readFileSync, accessSync, R_OK } from 'fs';
 import { join } from 'path';
 
+function readable(path) {
```

Comment:
`fs.existsSync` is deprecated and I think it's better to check that we can read the file: https://nodejs.org/api/fs.html#fs_fs_existssync_path
Is there a better place to put this utility function ?